assets/queries/cloudFormation/aws/access_key_not_rotated_within_90_days/metadata.json

```
@@ -4,9 +4,9 @@
   "severity": "MEDIUM",
   "category": "Secret Management",
   "descriptionText": "ConfigRule should enforce access keys to be rotated within 90 days.",
-  "descriptionUrl": "https://docs.amazonaws.cn/en_us/config/latest/developerguide/access-keys-rotated.html",
+  "descriptionUrl": "https://docs.aws.amazon.com//en_us/config/latest/developerguide/access-keys-rotated.html",
```

Hey! Just a small suggestion, your link works fine, but there's an extra "/" after amazon.com (i.e., https://docs.aws.amazon.com**//**en_us/...). 
It might be good to clean it up to keep things consistent, but totally up to you! 😄

```suggestion
  "descriptionUrl": "https://docs.aws.amazon.com/en_us/config/latest/developerguide/access-keys-rotated.html",
```

---

Oops! Fixed.